ui/public/checkBrowser.js

```diff
@@ -0,0 +1,27 @@
+function checkBrowser() {
+  if (!window.__supported_browsers__.test(navigator.userAgent)) {
+    var text
+    if (navigator.language.indexOf('zh') === 0) {
```

Comment:
IE6 ~ 10 even do not support NavigatorLanguage API, and I'm afraid it will be broken in such cases.

Reply:
Let me do some test.